extensions/wikia/Sitemap/SpecialSitemap_body.php

```diff
@@ -266,6 +277,7 @@ private function generateNamespace( Array $sitemapIndex = null, $forceUpdate = f
 				'page_namespace',
 				'page_title',
 				'page_touched',
+				'page_id',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "page_id" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)